### PR TITLE
Moved Bionics Systems Mod to base game and Crazy Cataclysm

### LINKED ISSUE #24780: Mod Inclusion Project: Bionics Systems
Use the following issue as a space to discuss the inclusion of:

Mod Ident: bio_mod
Name: Bionics Systems Mod

Your comment should include:

### Suggestions for induction
(what parts of the mod should be included, can be as simple as just an item, a location or the complete inclusion)

### Incompatibilities
(what parts of the mod should not be included, either some objects / items or entire mod is conflicting with DDA theme [ for whatever reason ])

---

edit:

Based upon feedback, Bionics Systems mod seems like a good fit. Therefore,

- [ ] - Add advanced form of Directional EMP CBM with the EMP Projectors
- [ ] - Add Autonomous Surgical Razors CBM as butchery equivalent to Integrated Toolset (potentially exclusive to Bionic Surgeon professions or other profession, could also be lab book-gated [like most serums])
- [ ] - Add Ionic Overload Generator CBM to Crazy Cataclysm OR to core but needs extreme balance/tuning
- [ ] - Add Bionic Surgeon profession with some tuning

All the aforementioned are based on suggestions in thread and not necessarily written in stone, but for any contributors that would like to induct the mod in, these are the suggestions to keep in mind at least.

### PR DESCRIPTION
**Pull request by Kelenius**
#### Summary
```SUMMARY: Mods "Moved Bionics Systems Mod to base game and Crazy Cataclysm."```

#### Purpose of change
Resolves #24780

#### Describe the solution
EMP projector was moved to the main game 1:1 and added to loot tables `bionics_sci` (scientist zombie corpses), `bionics_mil` (bunkers n stuff), `bionics_op` (bio-operator corpses). Notably, it's absent from `bionics_subs` (shocker brute corpses), which drops Directional EMP semi-commonly. Since CMB crafting is kill, I think that's a reasonable way to make it rarer.

Surgical razors renamed to "Autonomous Surgical Scalpels", because surgical razor is used to shave hair for surgery prep. They are now a passive tool that gives `"CUT", 1` (in line with cutting tools) `"CUT_FINE", 5` (highest currently is a regular scalpel, which has 3, but this is only advantageous for manual bionic removal so...) `"BUTCHER", 50` (higher than hunting knife with 30 but lower than powered saw with 60). It can be obtained from `bionics_sci` and `science`, so you'd need to get it from labs.

Ionic Overload Generator moved to Crazy Cataclysm mod 1:1.

Bionic Surgeon moved to the main game 1:1. There was a suggestion that he should be given a trait for autodoc operation, but nothing in its description says anything about installing bionics. It starts with some drugs and a rare bionic, but doesn't have a lot of survival skills or equipment, so I think 6 points is reasonable.

### CODE REVIEW

**Comment by Kelenius** [comment]
```12:06:25 Invalid Summary line, please provide a summary similar to the following:
12:06:25 SUMMARY: Features "Added ability to fly."```

Dude what. It's **right there**.

File: data/json/professions.json [hunk 1]
```diff
@@ -2895,5 +2895,34 @@
       "male": [ "boxer_shorts" ],
       "female": [ "bra", "panties" ]
     }
+  },
+  {
+    "type": "profession",
+    "ident": "bio_medic",
+    "name": "Bionic Surgeon",
+    "description": "As one of the top surgeons in the country, you were chosen for an augmentation program to expand the medical field. With your experties and augmentations, you can preform precise surgery with little assitance.",
```
Comment (mlangsdorf) [inline data/json/professions.json]: Last words should be "perform precise surgery with little assistance"
There should be one more space after the period between the sentences.

File: data/mods/CrazyCataclysm/crazy_items.json [hunk 2]
```diff
@@ -42,5 +42,39 @@
     "name": "scissor jack",
     "description": "A compact scissor jack used for lifting vehicles. It's outfitted with patented Aperture Science(tm) technology that lets it lift engines by teleporting itself beneath them.",
     "qualities": [ [ "LIFT", 1 ] ]
+  },
+  {
+    "id": "bio_chest_gun",
+    "copy-from": "bionic_general",
+    "type": "BIONIC_ITEM",
+    "name": "Ionic Overload Generator CBM",
+    "description": "A powerful ion energy generator is implanted on the user's chest.  Fires a powerful, ever-expanding energy blast.  The resulting blast ignites oxygen creating fires as it moves and an explosion on impact. Close range use is highly discouraged.",
```
Comment (mlangsdorf) [inline data/mods/CrazyCataclysm/crazy_items.json]: Needs one more space after the period before the last sentence.

File: data/json/items/bionics.json [hunk 3]
```diff
@@ -733,6 +742,15 @@
     "price": 800000,
     "difficulty": 11
   },
+  {
+    "id": "bio_surgical_razor",
+    "copy-from": "bionic_general",
+    "type": "BIONIC_ITEM",
+    "name": "Autonomous Surgical Scalpels CBM",
+    "description": "Implanted on the user's fingers is a system of surgical grade scalpels.  They will allow you to make automated precise cuts, and are useful as a butchering tool.",
```
Comment (mlangsdorf) [inline data/json/items/bionics.json]: Change the first sentence to "A system of surgical grade scalpels is implanted in the user's fingers."
Change the second sentence to "The user can make surgically precise cuts, which is also useful for butchery."

File: data/json/bionics.json [hunk 4]
```diff
@@ -314,6 +314,16 @@
     "occupied_bodyparts": [ [ "ARM_R", 1 ], [ "HAND_R", 2 ] ],
     "act_cost": 100
   },
+  {
+    "type": "bionic",
+    "id": "bio_emp_armgun",
+    "name": "EMP Projector",
+    "description": "A ranged emp generator system is implanted on the palm of your right hand and arm.  Fires super-concentrated electric pulses for a short distance.  Extremely effective against electronic targets but mostly useless otherwise.",
```
Comment (cake-pie) [inline data/json/bionics.json]: Please capitalize the abbreviation "EMP" here

File: data/json/items/bionics.json [hunk 5]
```diff
@@ -245,6 +245,15 @@
     "price": 720000,
     "difficulty": 5
   },
+  {
+    "id": "bio_emp_armgun",
+    "copy-from": "bionic_general",
+    "type": "BIONIC_ITEM",
+    "name": "EMP Projector CBM",
+    "description": "A ranged emp generator system is implanted on the palm of your right hand and arm.  Fires super-concentrated electric pulses for a short distance.  Extremely effective against electronic targets but mostly useless otherwise.",
```
Comment (cake-pie) [inline data/json/items/bionics.json]: here...

File: data/json/items/gun/bio.json [hunk 6]
```diff
@@ -20,6 +20,31 @@
     "ammo_effects": [ "PLASMA", "EXPLOSIVE", "FLAME" ],
     "flags": [ "NEVER_JAMS", "TRADER_AVOID" ]
   },
+  {
+    "id": "bio_emp_gun",
+    "type": "GUN",
+    "name": "EMP Projector",
+    "description": "A ranged emp generator system is implanted on the palm of the user's right hand and arm.  Fires super-concentrated electric pulses for a short distance.  Extremely effective against electronic targets but mostly useless otherwise.",
```
Comment (cake-pie) [inline data/json/items/gun/bio.json]: ... and here

**Comment by narc0tiq** [comment]
> Dude what. It's [the summary is] right there.

Yeah, I didn't fix it good: <https://github.com/CleverRaven/Cataclysm-DDA/pull/25367#issuecomment-418756736>

**Comment by Kelenius** [comment]
I really should have remade all the text instead of copying it over.

**Comment by Kelenius** [comment]
This should probably be added to https://github.com/CleverRaven/Cataclysm-DDA/projects/18

**Comment by Noctifer-de-Mortem** [comment]
@Kelenius If you still have issue with the summary, did you make it with three ` symbols around it rather than one?

**Comment by narc0tiq** [comment]
The last Jenkins build attempt was 8 days ago, maybe we should "Jenkins, rebuild" and see what it thinks now.

**Pull request merged**